apis/kueue/v1beta1/admissioncheck_types.go

```diff
@@ -85,11 +87,13 @@ type AdmissionCheckParametersReference struct {
 type AdmissionCheckStatus struct {
 	// conditions hold the latest available observations of the AdmissionCheck
 	// current state.
+	// This is limited to at most 64 separate conditions.
 	// +optional
 	// +listType=map
 	// +listMapKey=type
 	// +patchStrategy=merge
 	// +patchMergeKey=type
+	// +kubebuilder:validation:MaxItems=64
```

Comment:
This is just a question. Are there any recommendation guides to specify `64` for conditions?

Reply:
@JoelSpeed mentioned that I could use 16. I guessed here as I notice that Kueue used 64 in a few places.

Reply:
> No I just mean that I see 64 limit in a lot of places in our code base so I picked 64.

Do you indicate resourceGroup something? If yes, that is considered based on scheduling performance.
So, I think that we do not need to follow others.

Could we consider 16 limits for conditions as starting points?